mod/wikirate/lib/stylesheets/style_bars_and_boxes.scss

```diff
@@ -24,113 +26,142 @@ a.box-item {
   -moz-box-shadow: 0 9px 20px rgba(0, 0, 0, 0.2);
   box-shadow: 0 9px 20px rgba(0, 0, 0, 0.2);
 }
-
 // type specific colors
-
 $colorMap: (
   metric: $metric-color,
   company: $company-color,
   topic: $topic-color,
   source: $source-color,
   project: $project-color,
   answer: $answer-color,
-  research_group: $research-group-color
-);
-
-// border color for bars and boxes
+  research_group: $default-cardtype-color,
+  post: $default-cardtype-color
+); // border color for bars and boxes
 @mixin barbox($color) {
-  &.bar, &.bar-bottom {
+  &.bar,
+  &.bar-bottom {
     border-left-color: rgba($color, 0.2) !important;
 
     &:hover {
       border-left-color: $color !important;
     }
   }
+
   &.box {
     border-top-color: $color !important;
+
     .box-top a {
-      color: $color
+      color: $color;
     }
   }
-  &.bar:hover{
+
+  &.bar:hover {
     .bar-left a {
       color: $color;
     }
+
+    .bar-left .company-link a {
+      color: $company-color;
+    }
```

Comment:
can't we just add this once with `!important`?  if we put it in the mix-in, I think we'll get many copies of it.  Do we need that?